src/suggestion_engine/server/providers/closedTab.js

```diff
@@ -1,18 +1,15 @@
 import Fuse from 'fuse.js';
 import { isSakaUrl } from 'lib/url.js';
+import { filter } from 'rxjs/operator/filter';
 
 async function getAllSuggestions() {
-  // only show tabs not windows, TODO: show windows too
-  // .filter(
-  //   session =>
-  //     session.tab &&
-  //     session.tab.url !==
-  //       'chrome-extension://nbdfpcokndmapcollfpjdpjlabnibjdi/saka.html'
-  // )
-
   const sessions = await browser.sessions.getRecentlyClosed();
   const filteredSessions = [];
 
+  // TODO: This for loop is currently flagged by the airbnb eslint rules.
+  // See: https://github.com/airbnb/javascript/issues/1271
+  // Not disabling the rule as this might be fixable in the future using filter.
+  // This for loop is needed at the moment as a workaround since filter does not support async.
   for (const session of sessions) {
     const sakaUrl = await isSakaUrl(session.tab.url);
     if (session.tab && !sakaUrl) {
```

Comment:
browser.sessions.getRecentlyClosed() returns a list of closed tabs AND WINDOWs. This if statement filters the windows out. Ideally, it should be possible to restore windows, but I removed them to simplify the implementation.

Reply:
Good point, I will make a note to look into this. I skimmed through the documentation for [sessions](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/sessions/restore) and this doesn't seem too difficult to implement, there just needs to be a check to see if it is a tab or window and pass in the appropriate value.